Sources/Commands/SwiftTool.swift

```diff
@@ -55,8 +55,10 @@ struct TargetNotFoundDiagnostic: DiagnosticData {
     let targetName: String
 }
 
-private class ToolWorkspaceDelegate: WorkspaceDelegate {
-
+internal class ToolWorkspaceDelegate: WorkspaceDelegate {
```

Comment:
We generally avoid the `internal` keyword.

Reply:
Avoid as in omit?

Reply:
Yep